packages/core/admin/admin/src/components/NpsSurvey/tests/index.test.js

```diff
@@ -0,0 +1,195 @@
+import React from 'react';
+
+import { lightTheme, ThemeProvider } from '@strapi/design-system';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import NpsSurvey from '..';
+
+const localStorageMock = {
+  getItem: jest.fn(),
+  setItem: jest.fn(),
+  removeItem: jest.fn(),
+  clear: jest.fn(),
+};
+const originalLocalStorage = global.localStorage;
+
+const user = userEvent.setup();
+
+const setup = () =>
+  render(
+    <ThemeProvider theme={lightTheme}>
+      <NpsSurvey />
+    </ThemeProvider>
+  );
+
+describe('NPS survey', () => {
+  beforeAll(() => {
+    global.localStorage = localStorageMock;
+  });
+
```

Comment:
It is sufficient to mock this once using `jest.mock` and then use `jest.restoreAllMocks()` afterEach.

Reply:
Ah, sorry. I've made a mistake here und misunderstood something. Apologies - this should be fine. @simotae14